dev-libs/mauikit-accounts/mauikit-accounts-9999.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_QTHELP="true"
+KDE_ORG_CATEGORY="maui"
+KFMIN=5.82.0
+QTMIN=5.15.5
+inherit ecm kde.org
+
+DESCRIPTION="EDITME"
+
+LICENSE="GPL-3"
+SLOT="5"
+KEYWORDS=""
+IUSE=""
+
+DEPEND="
+	>=dev-qt/qtsql-${QTMIN}:5
+	>=dev-qt/qtcore-${QTMIN}:5
```

Comment:
dev-qt/qtcore is depended on by `ecm.eclass` already, you can drop it on all consumers.